torch/_dynamo/optimizations/training.py

```diff
@@ -40,7 +41,11 @@ def compiler_fn(gm: torch.fx.GraphModule, example_inputs):
 
         # TODO: stop monkeypatching here (without even cleaning up, UGH!)
         functorch.compile.config.use_functionalize = True
-        functorch.compile.config.use_fake_tensor = True
+
+        # torchxla does not work with FakeTensor
```

Comment:
hm, does AotAutograd still work with faketensor disabled? and do you require users to set USE_FAKE_TENSOR=0 when running xla?

Reply:
This works on trunk now. I can remove this part